templates/webmention-comment-form.php

```diff
@@ -1,13 +1,31 @@
+<?php
+/**
+ * Hook to add custom content before the webmention form added to the comment form.
+ */
+do_action( 'webmention_comment_form_template_start' );
```

Comment:
isn't `before`/`after` or `pre`/`post` the WordPress way to go?

Reply:
I'm happy to go with `before`/`after`. Done in c10bcaf86cc1e560c2606dcf44ac0721814a1304